### PR TITLE
Fix for QMOS not updating lat/lon ranges correctly. 

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

QMOS is broken on lon direction = positivewest on top of ranges being incorrect when new map files are loaded. This should fix the issue as described.  

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/3573

## Motivation and Context

We want the grid tool to work. 

## How Has This Been Tested?

Manually tested with all the projection files @lwellerastro posted as part of the test. This includes switching between postivewest/east and 180 to 360 domains while messing with options in the grid tool to see if grid still draws correctly. Everything seems to work. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/qisis/objs/MosaicSceneWidget/MosaicGridTool.cpp
Comment (krlberry) [inline isis/src/qisis/objs/MosaicSceneWidget/MosaicGridTool.cpp:810]: Needs a docstring.

**Review by krlberry — CHANGES_REQUESTED** [review]
:+1: The change, including the qt stuff looks good to me. (Though I also have very limited experience with signals/slots and qt. Some of the pre-existing code seems a little odd, but nothing you added.) Just requesting documentation.

**Review by AustinSanders — DISMISSED** [review]
Looks good to me

**Comment by Kelvinrr** [comment]
Once I get this in Ethan's hands and finds no issues + a re-review after the docstring change we can merge.

**Review by jessemapel — APPROVED** [review]

**Review by krlberry — APPROVED** [review]

**Review by AustinSanders — APPROVED** [review]

**Comment by Kelvinrr** [comment]
We got Ethan's blessing :+1:

**Pull request merged**
